quarkus-app/src/main/resources/templates/ProfileResource/profile.html

```
@@ -1,3 +1,4 @@
+{! Force cache invalidation - 2026-08-10 !}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)